### ISSUE TITLE
Update raw_high_lat with development

### ISSUE THREAD

**Issue by joeschoonover** [SWPC-IPE/WAM-IPE#116]
The raw_high_lat branch is becoming severely out of date with the development branch. We need to merge in development changes into raw_high_lat before continuing work.

**Comment by joeschoonover** [comment]
doing a straight merge with development is not a good idea. Instead, we should selectively checkout directories we know we want to carry over from development and the selectively patch files that require more granular merging using `git checkout --patch`

**Comment by joeschoonover** [comment]
In a series of commits I've checked out patches of code from development, finishing at hash id fc2df3989c60939a8a877705e08e1f0a58ac1a9c .Both cold starts and restarts work, but 1hr+1hr and 2hr simulations no longer match again. I've verified that all the necessary code to receive data from the WAM restart files is still in place and being executed. However, there does seem to be some peculiarities with how IPE is managing the time stepping in the raw_high_lat branch. A new issue will be opened dedicated to resolving the restart bug in the raw_high_lat branch
